docs/dashboards/introduction.md

```diff
@@ -132,6 +132,17 @@ Click on the **pencil** icon to enter dashboard edit mode, hover over the questi
 
 ![Visualization settings](images/visualization-settings.png)
 
+### Hiding a card when it doesn't return results
+
+One neat thing to call out: if you have a question card that rarely returns results, but you still want to include it in your dashboard because you want to know when it _does_ return results, you can tell Metabase to hide the card unless it returns at least one row of data.
```

Comment:
```suggestion
One neat thing to call out: if you have a question card that rarely returns results, but you still want to include it in your dashboard because you want to know when it _does_ return results, you can tell Metabase to hide the card until it returns at least one row of data.
```